### PR TITLE
garg2021re: [Re] Training Binary Neural Networks using the Bayesian Learning Rule

### PR DESCRIPTION
**Pull request by prateekstark**
Resolved all compilation issues.

### CODE REVIEW

**Comment by koustuvsinha** [comment]
Thanks, compilation works on my end. However, will it be possible to shrink Tables 1 and 2? Their width is overflowing the page margins. (Tip: try `\tiny` font)

**Comment by prateekstark** [comment]
Updates the code and fixed the table sizes. Please let us know further required changes.

**Comment by koustuvsinha** [comment]
Thanks, looks good to me! :)

**Comment by koustuvsinha** [comment]
@jzf2101 can you review the contents of the paper?

**Comment by asdas1505** [comment]
@koustuvsinha  Error: fatal: No url found for submodule path 'RC-2020' in .gitmodules
Error: The process '/usr/bin/git' failed with exit code 128 

The above error might be coming up because of modified .gitmodules as suggested in #14

**Comment by jzf2101** [comment]
I think this paper isn't ready yet since it can't build?

**Comment by asdas1505** [comment]
@jzf2101 Build error might be coming because we modified .gitmodules as discussed in issue #14. We have made the required changes, can you please try building it again?

**Comment by koustuvsinha** [comment]
I did confirm before adding the latex-ok label that build was working on my end. @prateekstark apologies for the confusion.

**Comment by koustuvsinha** [comment]
> I think this paper isn't ready yet since it can't build?

@jzf2101 I had reviewed this paper before and build works on our end. Please ignore the warning from the Github workflow, as it was triggered due to another project at that time.

**Comment by koustuvsinha** [comment]
@prateekstark It seems your PR is affecting the files in other directories. Can you clean your commit such that only the files in your folder are affected? Thanks!

**Comment by prateekstark** [comment]
Thanks, @koustuvsinha for pointing that out. I guess the problem was occurring because I tried to run build_all.sh on my local system but now I've cleaned up the repo and it has no conflicts with the base branch.
